python/mxnet/gluon/block.py

```diff
@@ -1149,30 +1156,43 @@ def hybridize(self, active=True, backend=None, backend_opts=None, clear=True, **
             The name of backend, as registered in `SubgraphBackendRegistry`, default None
         backend_opts : dict of user-specified options to pass to the backend for partitioning, optional
             Passed on to `PrePartition` and `PostPartition` functions of `SubgraphProperty`
-        clear : clears any previous optimizations
-        static_alloc : bool, default False
+        clear : bool, default True
```

Comment:
are we changing the clear default for optimize_for, but leaving the default to True for hybridize? That seems to make sense, existing behavior for hyridize was to clear on every call so that makes sense. Just checking

Reply:
That's correct!